### DISCUSSION TITLE
It should be possible to change the default core for the entire emulator, not just the selected game.

### DISCUSSION THREAD

**Discussion by ruaxbk** [OnionUI/Onion#1662] [Ideas]
here is an option for "Game core" but none for "Emulator core".

When deciding on which core to use for launching a game, the order of precedence would be: 
Game core > Emulator core > Default core.

**Comment by adamcroxton** [comment]
Would love to see this.  A per game and a per console core option.  I generally use the default and have little issues.  If a game isn't running right I will just change it.  But it would be very convenient none the less.

**Reply by oherau** [reply]
So far you can do this by modifying the launch.sh script of the system you want to modify.
Remember to make a backup just in case you want to revert it.
